core/controllers/dashboard_test.py

```diff
@@ -463,6 +463,117 @@ def test_no_explorations(self):
         self.assertEqual(response['explorations_list'], [])
         self.logout()
 
+    def test_view_dashboard_no_explorations(self):
```

Comment:
I was thinking in the way like a creator first creates and than logs out and after that a viewer comes in and visits the dashboard and than logs out.But as you mentioned earlier i will do it more simply without checking the workflow twice.